packages/rpms/SPECS/wazuh-manager.spec

```diff
@@ -584,10 +575,8 @@ if [ -f %{_localstatedir}/tmp/wazuh.restart ]; then
   if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1 ; then
     systemctl daemon-reload > /dev/null 2>&1
     systemctl restart wazuh-manager.service > /dev/null 2>&1
-  elif command -v service > /dev/null 2>&1 ; then
+  else command -v service > /dev/null 2>&1 ; then
```

Comment:
There is only an else part, I think it is correct as it already is.

Reply:
But the else clause shouldn't have a conditional check like `command -v service > /dev/null 2>&1 ; then`. To check if the service command exists, you should use elif. This is important because the command `service wazuh-manager restart > /dev/null 2>&1` relies on service being available.